rosa_architecture/rosa_policy_service_definition/rosa-sre-access.adoc

```diff
@@ -16,15 +16,26 @@ For a list of the available subprocessors, see the link:https://access.redhat.co
 
 include::modules/sre-cluster-access.adoc[leveloffset=+1]
 include::modules/rosa-red-hat-support-access.adoc[leveloffset=+1]
+ifdef::openshift-rosa,openshift-rosa-hcp[]
+[role="_additional-resources"]
+.Additional resources
+* For more information, see xref:../../support/approved-access.adoc#approved-access[Approved Access].
+
+* For more information, see xref:../../authentication/using-rbac.adoc#default-roles_using-rbac[Default cluster roles]
+endif::openshift-rosa,openshift-rosa-hcp[]
+
+
 include::modules/rosa-customer-access.adoc[leveloffset=+1]
 include::modules/rosa-access-approval-review.adoc[leveloffset=+1]
 include::modules/how-service-accounts-assume-aws-iam-roles-in-sre-owned-projects.adoc[leveloffset=+1]
 
-ifndef::openshift-rosa-hcp[]
+ifdef::openshift-rosa,openshift-rosa-hcp[]
 [role="_additional-resources"]
 .Additional resources
 
 * For more information about the AWS IAM roles used by the cluster Operators, see xref:../../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-operator-roles_rosa-sts-about-iam-resources[Cluster-specific Operator IAM role reference].
 
 * For more information about the policies and permissions that the cluster Operators require, see xref:../../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-account-wide-roles-and-policies-creation-methods_rosa-sts-about-iam-resources[Methods of account-wide role creation].
-endif::openshift-rosa-hcp[]
\ No newline at end of file
+
+* For more information, see xref:../../support/approved-access.adoc#approved-access[Approved Access].
+endif::openshift-rosa,openshift-rosa-hcp[]
```

Comment:
Since you have the new Additional resources section closer to the reference above, I think you can get rid of the Approved Access item on this list.

Then:
- If the docs were as intended before you started, revert the conditional to its former state (basically, this whole section would return to how you found it)
- If the docs were wrong, and these existing refs should have been in HCP but were mistakenly excluded, the same applies in the new Additional resources list: If the only places this assembly is used are ROSA and ROSA HCP, I don't think you need the conditional at all. But again, I am not sure how to verify where the assembly is pulled in.